lib/resty/healthcheck.lua

```diff
@@ -1436,13 +1416,49 @@ function _M.new(opts)
     worker_events:poll()
   end
 
-  -- start timers
+  -- turn on active health check
   local ok, err = self:start()
   if not ok then
     self:stop()
     return nil, err
   end
 
+  -- if active checker is needed and not running, start it
+  if (self.checks.active.healthy.active or self.checks.active.unhealthy.active)
+    and active_check_timer == nil then
+
+    self:log(DEBUG, "starting active check timer")
+    local err
+    active_check_timer, err = resty_timer({
+      recurring = true,
+      interval = 0.01, -- evaluate active checks every 0.01s
```

Comment:
This is very high frequency. we should have a lower frequency. Although the interval for the healthcheckers is a float, and can be very small, I think those are very rare.

Reply:
Good point. I reduced the frequency to every 0.1s.